### PR TITLE
Issue templates tweaks

### PR DESCRIPTION
**Pull request by dawidd6**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

- new `Bottle Request` type of issue, so users can easily request a bottle
- bring us more inline with upstream

### CODE REVIEW

**Comment by issyl0** [comment]
I like it. Would any of these fit in Homebrew/.github so they propagate to everywhere? I guess these are Linux-specific? That's fine if so, and better!

**Comment by dawidd6** [comment]
Yep, bottle requests are linux specific, I guess.

I'm gonna rebase this PR on top of latest master after merging upstream and make further changes.

File: .github/ISSUE_TEMPLATE/Reproducible-Bug-Report.md [hunk 1]
```diff
@@ -1,21 +0,0 @@
----
```
Comment (dawidd6) [inline .github/ISSUE_TEMPLATE/Reproducible-Bug-Report.md]: This file is now `bug.md` from upstream.

**Comment by dawidd6** [comment]
Can't really test it, unless this is present on master. So I'll allow myself to merge this already, test it live and fix later if needed.

**Pull request merged**